src/Microsoft.AspNet.SignalR.Core/Infrastructure/PerformanceCounterManager.cs

```diff
@@ -438,6 +438,14 @@ public IPerformanceCounter LoadCounter(string categoryName, string counterName,
             // and when they are thrown. 
             try
             {
+                if (!PerformanceCounterCategory.Exists(categoryName))
```

Comment:
I added the check as I have some developers on my team who disable the "just my code" option from time to time. This exception was causing them some grief. I don't believe that having the counters not installed is an unexpected scenario. Help me understand why the check is bad?